pycheribuild/projects/cross/busybox.py

```diff
@@ -181,11 +181,23 @@ def setup(self) -> None:
         self.make_args.set(CONFIG_PREFIX=self.install_dir / "rootfs")
         self.make_args.set(MUSL_HOME=self.install_dir)
         self.make_args.set(KHEADERS=self.install_dir / "usr/include/")
-        self.make_args.set(CLANG_RESOURCE_DIR=self.install_dir / "include/lib")
-        # https://git.morello-project.org/morello/morello-busybox/-/merge_requests/1
-        self.make_args.set(CFLAGS="-Wno-error=int-conversion")  # for editors/vi
+        self.make_args.set(CLANG_RESOURCE_DIR=self.install_dir / "lib")
         self.add_configure_vars(CC=self.CC)
 
+        # Override Morello's busybox custom Makefile; it uses its own flags assuming compiler-rt
+        # is in the compiler resource directory (which we don't do in cheribuild)
+        self.make_args.set(
```

Comment:
Would the need for this go away if we call super().setup()?

Reply:
Everything else went away now, except for this LD override. Please see the latest push and let me know what you think and whether we can do something about it in the base class. Anyway, this Morello child class is ought to go once and we can hopefully use the base when we move to Codasip's busybox (or do our own)